tests/test_models/test_sarimax_model.py

```diff
@@ -38,6 +38,23 @@ def test_sarimax_forecaster_run_with_reg(example_reg_tsds):
     assert len(res) == 14
 
 
+def test_sarimax_forecaster_run_with_reg_case2(example_reg_tsds):
```

Comment:
May be to name it like `test_sarimax_forececaster_run_with_reg_custom_order`. I also think that forecaster here is a little bit irrelevant, because it is a model (but we should look at the other tests first).

Reply:
```suggestion
def test_sarimax_forececaster_run_with_reg_custom_order(example_reg_tsds):
```